### PR TITLE
F#src 2584 add cyberglove_and_aws

### PR DESCRIPTION
**Pull request by ToivoHn**

### CODE REVIEW

File: ansible/roles/aws/tasks/main.yml [hunk 1]
```diff
@@ -1 +1,17 @@
...
+- name: Install AWS CLI
+  pip:
+    requirements: "{{ role_path }}/data/requirements.txt"
+  become: true
+
+- name: Copy shadow_upload.sh to the docker container
+  shell: "docker cp {{ role_path }}/files/shadow_upload.sh {{ container_name }}:/usr/local/bin/shadow_upload.sh"
+
+- name: Copy customer.key to the docker container
+  shell: "echo {{ customer_key }} > {{ role_path }}/files/customer.key; docker cp {{ role_path }}/files/customer.key {{ container_name }}:/usr/local/bin/customer.key"
```
Comment (AndriyPt) [inline ansible/roles/aws/tasks/main.yml]: it is not that good idea to change file system of existing role. Please use /tmp for that

File: ansible/roles/products/hand_e/cyberglove/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+- name: Create Cyberglove executable launch script
+  template:
+    src: templates/scripts/launch_cyberglove.j2
+    dest: "{{ shadow_hand_launcher_folder }}/launch_cyberglove.sh"
+    mode: '755'
+
+- name: Create Cyberglove desktop icon
+  template:
+    src: ../desktop-icons/templates/desktop_labels/desktop_icon.j2
```
Comment (AndriyPt) [inline ansible/roles/products/hand_e/cyberglove/tasks/main.yml]: using file from different role. Seems really bad idea. You are breaking encapsulation of role. Please add big TODO here to fix it.

File: ansible/roles/products/hand_e/docker-container/templates/scripts/setup_hand.j2 [hunk 3]
```diff
@@ -4,7 +4,22 @@ source /home/user/projects/shadow_robot/base/devel/setup.bash
 roscd sr_ethercat_hand_config
 git fetch
 git checkout {{ config_branch }}
+if {{ use_cyberglove | lower }}; then
```
Comment (AndriyPt) [inline ansible/roles/products/hand_e/docker-container/templates/scripts/setup_hand.j2]: Add TODO to thing how make it better

File: ansible/roles/aws/tasks/main.yml (hunk 1 above)
Comment (ToivoHn) [reply ansible/roles/aws/tasks/main.yml]: Fixed in the last commit

File: ansible/roles/products/hand_e/cyberglove/tasks/main.yml (hunk 2 above)
Comment (ToivoHn) [reply ansible/roles/products/hand_e/cyberglove/tasks/main.yml]: Fixed in the last commit

File: ansible/roles/products/hand_e/docker-container/templates/scripts/setup_hand.j2 (hunk 3 above)
Comment (ToivoHn) [reply ansible/roles/products/hand_e/docker-container/templates/scripts/setup_hand.j2]: Fixed in the last commit

**Pull request merged**